README.md

```diff
@@ -19,9 +19,9 @@ PBS Professional® software optimizes job scheduling and workload management in
 
 ## Community and Ways to Participate
 
-PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct).
+PBS Professional is a community effort and there are a variety of ways to engage, from helping answer questions to benchmarking to developing new capabilities and tests.  We value being aggressively open and inclusive, but also aggressively respectful and professional (see our [Code of Conduct](https://pbspro.atlassian.net/wiki/display/PBSPro/Code+of+Conduct)).
```

Comment:
I suggest ending the sentence after "professional" and start a new sentence without parentheses stating...
Refer to our code of conduct page http://...